src/metabase/api/notify.clj

```diff
@@ -39,4 +39,4 @@
   {:success true})
 
 
-(api/define-routes)
+(api/define-routes routes)
```

Comment:
How were all of these working before?

Reply:
I changed the macro to have you explicitly specify the name that gets deffed. Previously it would always define `routes`. It's a little more obvious this way and easier to make clj-kondo stop complaining